src/main/java/org/spongepowered/api/command/CommandCallable.java

```diff
@@ -79,6 +79,20 @@
     boolean testPermission(CommandSource source);
 
     /**
+     * Test whether this command is allowed to be executed by the specified
```

Comment:
@simon816 Ok, my bad. If it's usually in that form then it should be added to the format guide.